### PR TITLE
update semgrep and deps

### PR DESCRIPTION
**Pull request by creviera**
# Description

Update test dependencies

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --allow-unsafe --generate-hashes --output-file=test-requirements.txt test-requirements.in
+#    pip-compile --generate-hashes --output-file=test-requirements.txt test-requirements.in
```
Comment (emkll) [inline test-requirements.txt:5]: is there a reason why `--allow-unsafe` in the pip-compile command here? We use that parameter to pin `pip` and `setuptools` in the requirements files.

Comment (emkll) [reply test-requirements.txt:5]: confirmed out of band with @creviera that this is not strictly required.

**Review by emkll — APPROVED** [review]
LGTM, thanks. Added https://github.com/freedomofpress/securedrop-export/commit/36b95b97e0680c2789e49aa79f0c7fc8314a16e9 in the spirit of the changes you already introduced in https://github.com/freedomofpress/securedrop-client/pull/1253/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R120

**Comment by creviera** [comment]
good catch @emkll! https://github.com/freedomofpress/securedrop-export/pull/75#pullrequestreview-649528993

**Pull request merged**